__tests__/app.test.js

```diff
@@ -305,4 +305,78 @@ describe("PATCH /api/reviews/:review_id", () => {
     })
 })
 
+describe("GET /api/reviews", () => {
+    it("200: Endpoint can accept additional queries. If 'category' is passed then endpoint should only return reviews with the corresponding category", () => {
+        return request(app)
+        .get("/api/reviews?category=dexterity")
+        .expect(200)
+        .then((response) => {
+            const returnedReview = response.body.reviews
+            expect(returnedReview.length).toBe(1)
+            returnedReview.forEach((review) => {
+                expect(review.category).toBe("dexterity")
+                expect(review.category).not.toBe("euro game")
+                expect(review.category).not.toBe("social deduction")
+            })
+        })
+    })
+    it("400: When passed a 'category' query that does not exist, endpoint returns a bad request error", () => {
```

Comment:
Here the category doesn't exist in the database currently, but could be added later. This should be a 404 'Not found' Instead of a 400.